src/main/resources/js/jira-octane-plugin-admin.js

```diff
@@ -261,12 +261,16 @@
         }
 
         //validate
-        var nameValue = $("#name").attr("value");
+        var nameValue = $("#name").attr("value").trim();
+        var locationValue = $("#location").attr("value").trim();
+        var clientIdValue = $("#clientId").attr("value").trim();
```

Comment:
maybe add them in backend as well, not sure what happens if you save a space config with only whitespaces through REST API and then open the ui